proto/ligato/linux/interfaces/interface.proto

```diff
@@ -52,11 +58,19 @@ message Interface {
 
         // TAP_TO_VPP-specific configuration
         TapLink tap = 21;
+
+        // VRF_DEVICE-specific configuration
+        VrfDevLink vrf_dev = 22;
     };
 
     // Configure/Resync link only. IP/MAC addresses are expected to be configured
     // externally - i.e. by a different agent or manually via CLI.
     bool link_only = 9;
+
+    // VRF to which this interface should be assigned to.
+    // The field should reference the logical name of a VRF device (i.e. instance of this Interface model with
+    // VRF_DEVICE type).
+    string vrf = 10;
```

Comment:
I'm not sure about the naming `vrf` here.. we use `*_interface`/`*interfaces` practically everywhere else where interface is references. Using `vrf_interface` is more consistent and possibly clearer to the users.

Reply:
What about `vrf_master_interface` or shorter variant `vrf_master_if`?
Just to make it more clear that what is being referenced is not some random interface inside a VRF, but the interface implementing VRF as a whole.
(master/slave terminology is not very popular these days but that's how they call it in Linux - `ip link set dev eth1 master vrf-blue` is said to `enslave` `eth1` under `vrf-blue` interface/device)

Reply:
Yea, I think that's okay.